### DISCUSSION TITLE
Is improve_contrast supposed to be stored as a string?

### DISCUSSION THREAD

**Discussion by mrmees** [blakeblackshear/frigate#14223] [Ask A Question] [answered]
The improve_contrast value is currently stored as a string instead of a boolean when selecting that option via the settings -> Motion Tuner interface.  Is that expected behavior?

**Comment by hawkeye217** [comment, marked as answer]
This is a duplicate of https://github.com/blakeblackshear/frigate/discussions/13797

The backend sees the value correctly regardless of the quotes. However, this has already been improved in dev and will be available in Frigate 0.15.

**Comment by NickM-27** [comment]
It's not an issue, but it has been fixed for the next version
